### ISSUE TITLE
Make App Middleware stable in v1.12

### ISSUE THREAD

**Issue by msfussell** [dapr/dapr#6808]
App Middleware was introduced in the v1.9 and needs to be made stable in v1.12 after 3 releases
See https://docs.dapr.io/operations/support/support-preview-features/
And feature description here https://docs.dapr.io/operations/components/middleware/#app-middleware
This is a tracking issues to determine the remaining work required

**Comment by ItalyPaleAle** [comment]
I don't believe there is much left to do here. We have E2E tests as well.

**Comment by yaron2** [comment]
Agree, this looks to be about updating docs now. Moving to docs repo
